src/renderer/screens/Editor/ImportExportDialog.js

```diff
@@ -65,6 +85,10 @@ export const ImportExportDialog = withSnackbar(props => {
       });
   }
 
+  function loadDefault(layout) {
+    setData(JSON.stringify(layout));
+  }
```

Comment:
To be future-proof, would it be possible to load from a file instead? Like, automatically construct the filename based on the active keyboard (a'la the default/experimental firmware filenames)? That way, we wouldn't need to import them directly, and would make it easier to add defaults for other keyboards than the Model01 later down the road.

Reply:
Hmm... This does give me another idea for this that will integrate a solution for both of these comments. Will work on this and update PR when it's ready.